backend/Program.cs

```diff
@@ -67,7 +69,11 @@ public static async Task Main(string[] args)
                 var userManager = scope.ServiceProvider.GetRequiredService<UserManager<User>>();
                 await SampleUsers.CreateAdminAsync(userManager);
                 await SampleUsers.CreateUsersAsync(userManager);
+                
+                var seeder = scope.ServiceProvider.GetService<SeedSpecies>();
```

Comment:
I think `seeder` should be renamed to be renamed to something like `speciesSeeder` that way we know which type of data is being seeded, so when more seed data is added it will be clear

Reply:
Done!